src/components/EmojiPicker/EmojiPickerMenu/types.ts

```diff
@@ -0,0 +1,25 @@
+import type {Emoji} from '@assets/emojis/types';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type EmojiPickerMenuProps = {
+    /** Function to add the selected emoji to the main compose text input */
+    onEmojiSelected: (emoji: string, emojiObject: Emoji) => void;
+
+    activeEmoji?: string;
+};
+
+type RenderItemProps = {
+    item: Emoji & {
+        spacer?: boolean;
+
+        header?: boolean;
+    };
+
+    target: string;
+
+    index?: number;
+};
```

Comment:
NAB: Remove empty lines, or better add descriptions to each prop
```suggestion
type RenderItemProps = {
    item: Emoji & {
        spacer?: boolean;
        header?: boolean;
    };
    target: string;
    index?: number;
};
```